### PR TITLE
Updated mix indentation, issue #456

### PR DESCRIPTION
**Pull request by lisaychuang**
**What this PR does / why we need it**:

Updated [openebs/k8s/demo/Vagrantfile](https://github.com/openebs/openebs/blob/74aaee06d5b737b21145834958608150ecfa48df/k8s/demo/Vagrantfile) following [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide) indentation best practice:

* Use two spaces per indentation level (aka soft tabs). No hard tabs.
* Align the parameters of a method call if they span more than one line. When aligning parameters is not appropriate due to line-length constraints, single indent for the lines after the first is also acceptable.
* If multiple lines are required to describe the problem, subsequent lines should be indented three spaces after the # (one general plus two for indentation purpose).

**Which issue this PR fixes** 
This PR is for issue #456 -- mixed indentation.  

Please review @kmova

### CODE REVIEW

File: k8s/demo/Vagrantfile [hunk 1]
```diff
@@ -305,11 +307,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       mem = MH_MEM
 
       if ((deploy_Mode.to_i == DEPLOY_MODE_DEDICATED.to_i) || \
-        (deploy_Mode.to_i == DEPLOY_MODE_NONE.to_i))
+          (deploy_Mode.to_i == DEPLOY_MODE_NONE.to_i))
         config.vm.define hostname do |vmCfg|
           vmCfg.vm.box = "openebs/openebs-0.2"
-	        vmCfg.vm.provider "virtualbox" do |vb|
-           vb.customize ["modifyvm", :id, "--uartmode1", "file", File.join(Dir.pwd, "openebs-openebs-0.2-cloudimg-console.log")]
+	      vmCfg.vm.provider "virtualbox" do |vb|
```
Comment (kmova) [inline k8s/demo/Vagrantfile]: Line 313 to 315 -- needs to be indented properly.

File: k8s/demo/Vagrantfile [hunk 2]
```diff
@@ -251,8 +252,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         (deploy_Mode.to_i == DEPLOY_MODE_NONE.to_i))     
         config.vm.define hostname do |vmCfg|
           vmCfg.vm.box = "openebs/openebs-0.2"
-	        vmCfg.vm.provider "virtualbox" do |vb|
-           vb.customize ["modifyvm", :id, "--uartmode1", "file", File.join(Dir.pwd, "openebs-openebs-0.2-cloudimg-console.log")]
+	      vmCfg.vm.provider "virtualbox" do |vb|
```
Comment (kmova) [inline k8s/demo/Vagrantfile]: Line 255 to 256 -- needs to be indented properly.

**Comment by kmova** [comment]
@lisaychuang - do you use any ruby lint tool, that can verify/warn the non-compliance code?

**Comment by lisaychuang** [comment]
Strange, I use ruby-linter on VS code. The sections you pointed out were accurately indented on my editor...

alt text

I'll update the sections now.

File: k8s/demo/Vagrantfile [hunk 3]
```diff
@@ -252,7 +252,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         (deploy_Mode.to_i == DEPLOY_MODE_NONE.to_i))     
         config.vm.define hostname do |vmCfg|
           vmCfg.vm.box = "openebs/openebs-0.2"
-	      vmCfg.vm.provider "virtualbox" do |vb|
+	  vmCfg.vm.provider "virtualbox" do |vb|
```
Comment (kmova) [inline k8s/demo/Vagrantfile]: this github diff.. says that 255 and 313 may need a space.

**Comment by lisaychuang** [comment]
Still looks aligned on my screen....  should I update it anyway?

alt text
https://ibb.co/meu2rG

alt text
https://ibb.co/i3xYyw

**Comment by lisaychuang** [comment]
Thanks @kmova . What a strange issue, I've never encountered that before.

**Comment by nitisuryawanshi** [comment]
@lisaychuang Thank you so much for your PR contribution! We would like to mention you in our tweets, thanking you for your contribution. Can you please send me your twitter handle on this mail ID <niti.suryawanshi@openebs.io> ?

**Comment by lisaychuang** [comment]
awww thanks @nitisuryawanshi , I've just sent you an email 👍

**Pull request merged**